lib/Doctrine/DBAL/Schema/PostgreSqlSchemaManager.php

```diff
@@ -378,6 +378,12 @@ protected function _getPortableTableColumnDefinition($tableColumn)
             case 'year':
                 $length = null;
                 break;
+            case 'timestamp':
+            case 'timestamptz':
+                if ($tableColumn['default'] == 'now()') {
```

Comment:
It is kind of subtle; the tests will create a new table using the mapping of `CURRENT_TIMESTAMP`, and then the schema manager will call `listTableDetails()` on the database itself, which will report the column as having a default of `now()` (e.g. `not null default now()`). So the actual change in this PR is recognizing that "now()" is the same as the platform's `getCurrentTimestampSQL()`.

The reason the tests must use `CURRENT_TIMESTAMP` and not `NOW()` is that it must match what is in `AbstractPlatform::getCurrentTimestampSQL()`.